### ISSUE TITLE
Please support spyder 3.x

### ISSUE THREAD

**Issue by picca** [PierreRaybaut/guidata#38]
Idem guiqwt ;)

**Comment by stonebig** [comment]
I would suppose the problem comes from "site-package\spyderlib" that has becomed "site_package\spyder":

````
# Guidata (Python library generating graphical user interfaces for easy dataset editing and display)
from guidata import tests; tests.run()
````

````
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-dc950e79b631> in <module>()
      1 # Guidata (Python library generating graphical user interfaces for easy dataset editing and display)
----> 2 from guidata import tests; tests.run()

C:\WinPython\basedir35\buildQt5\winpython-64bit-3.5.x.3\python-3.5.2.amd64\lib\site-packages\guidata\tests\__init__.py in run()
     14     """Run guidata test launcher"""
     15     import guidata
---> 16     from guidata.guitest import run_testlauncher
     17     run_testlauncher(guidata)
     18 

C:\WinPython\basedir35\buildQt5\winpython-64bit-3.5.x.3\python-3.5.2.amd64\lib\site-packages\guidata\guitest.py in <module>()
     15 import subprocess
     16 
---> 17 from spyderlib.widgets.sourcecode.codeeditor import CodeEditor
     18 
     19 # Local imports

ImportError: No module named 'spyderlib'
````

**Comment by stonebig** [comment]
a "blind" 17 replace_all with notepad++ on "site_package\guidata" seems to suffice:
- replace "spyderlib"
- per "spyder"
- on all .*.py" objects of guidata

... well, it's not more compatible with Spyder 2 like that, but it works

**Comment by stonebig** [comment]
I may propose a patch

**Comment by stonebig** [comment]
it seem I can pat "live", but I can't create a proper wheel, "pip install fails":

````
  File "C:\WinPython\$results\winpython-64bit-3.4.4.4Qt5b1\python-3.4.4.amd64\lib\distutils\command\install.py", line 306, in finalize_options
    self.expand_basedirs()
  File "C:\WinPython\$results\winpython-64bit-3.4.4.4Qt5b1\python-3.4.4.amd64\lib\distutils\command\install.py", line 473, in expand_basedirs
    self._expand_attrs(['install_base', 'install_platbase', 'root'])
  File "C:\WinPython\$results\winpython-64bit-3.4.4.4Qt5b1\python-3.4.4.amd64\lib\distutils\command\install.py", line 467, in _expand_attrs
    val = subst_vars(val, self.config_vars)
  File "C:\WinPython\$results\winpython-64bit-3.4.4.4Qt5b1\python-3.4.4.amd64\lib\distutils\util.py", line 204, in subst_vars
    raise ValueError("invalid variable '$%s'" % var)
ValueError: invalid variable '$'results''
Warning: `HTML Help Workshop` is required to build CHM documentation file
warning: no files found matching '*.inc' under directory 'guidata'
warning: no files found matching '*.pot' under directory 'guidata'
no previously-included directories found matching 'guidata\tests\dist'
warning: sdist: standard file not found: should have one of README, README.rst, README.txt

**Comment by PierreRaybaut** [comment]
Fixed in b1d03af
